### ISSUE TITLE
Loading hangs on nuxtServerInit get to localhost base url

### ISSUE THREAD

**Issue by robyedlin** [nuxt/nuxt.js#3133]
I am getting "waiting for localhost" for 20-30 seconds when reloading a page that has an axios get to a localhost api base url in the nuxtServerInit function.

Direct curl to localhost api base url is instant. Seems to be an issue with nuxtServerInit.

Anyone else having this issue during development?

**Comment by xloading** [comment]
Have exactly the same problem, reproduction link is here: https://github.com/nuxt/nuxt.js/issues/3463
@robyedlin did you resolve it somehow?

**Comment by robyedlin** [comment]
Not really. Something to do with chrome sockets being tied up. Clearing out the sockets and then using incognito mode seems to work.

What tech stack are you using as your api server? How is it being served? I'm using python flask in a separate venv.

**Comment by xloading** [comment]
Hi Rob!

I'm using PHP framework (Yii2) for my REST API, it is hosted on the same server and served through Nginx/Apache. Don't think this is what is causing the issue though...

The problem is that I'm facing this on production as well, therefore shouldn't be caused by Chrome sockets as well.

Do you use router-module?
